win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -1019,8 +1019,7 @@ SetDIBitsToDevice(
 
     if (pConvertedInfo->bmiHeader.biHeight < 0)
     {
-        if (pConvertedInfo->bmiHeader.biHeight < -MaxSourceHeight || 
-            (YDest >= 0 && src_y < -ScanLines))
+        if (pConvertedInfo->bmiHeader.biHeight < -MaxSourceHeight || YDest >= 0)
```

Comment:
> no longer necessary

Do we know when/why that changed?

Reply:
When I started to ask stupid questions.

Reply:
After review, this appears to be an intermediate step which only affected the function's return value.
It had no effect of the displayed graphics. I just missed cleaning it up. It did not provide the improvement intended.

Reply:
Alright, then.
I suggest the PR/commit should explain that too/instead.